lms/static/styles/_frontend-shared.scss

```diff
@@ -18,4 +23,23 @@
   &--tall {
     height: min(90vh, 25rem);
   }
+
+  // This application does not have a CSS reset. Heading elements still retain
+  // browser defaults, including a margin that is problematic in Dialog and
+  // Modal headings.
+  // TODO: Re-assess after implementing typography/reset strategy
+  & h2 {
+    margin: 0;
+  }
+
+  & > .LMS-Scrollbox {
+    // Dialogs apply vertical spacing such that their immediate children are
+    // evenly spaced. This is a good thing, however, there is a certain edge case
+    // when we have a Scrollbox and we want it to scroll up flush against the
+    // bottom border of the Dialog header. In this case we want to explicitly
+    // disable the top margin so that it can be up against the header.
+    margin-top: 0;
+    // The Scrollbox border in this case is distracting.
+    border: none;
+  }
```

Comment:
I'm not thrilled, but I've been mucking around with this long enough that I want to set it down for the moment. This situation represents a true edge case in terms of selector specificity (i.e. that's why I didn't handle it in styles for the affect component). I will be revisiting, which is why I'm quarantining this in this SASS module.